tests/test_ensembl.py

```diff
@@ -0,0 +1,58 @@
+import json
+import os
+import pickle
+import re
+
+from httmock import all_requests, HTTMock
+import pytest
+
+
+@pytest.fixture
+def ensembl_protein_id():
+    return "ENSP00000354687"
+
+
+@all_requests
+def ensembl_mock(url, request):
+    match = re.match(r'.*/(.*?)$', url.path)
+
+    if match is None:
+        raise ValueError(f"URL {url} doesn't end in an Ensembl ID. Is this a "
+                         "valid Ensembl REST URL?")
+
+    ensembl_id = match.group(1)
+
+    try:
+        filename = os.path.join(pytest.config.rootdir,
+                                f'khtools/tests/data/{ensembl_id}.pkl')
+        with open(filename, 'rb') as testing_file:
+            testing_data = pickle.load(testing_file)
+    except FileNotFoundError:
+        raise ValueError(f"Testing file for transcript {ensembl_id} "
+                         "not found")
+
+    assert testing_data['ensembl_id'] == ensembl_id
+    print(f"Loaded test data for {ensembl_id}")
+    print(f"Test data generated on {testing_data['time_generated']}")
+
+    if 'sequence' in url.path:
+        return testing_data['sequence_response']
+    elif 'lookup' in url.path:
+        return testing_data['lookup_response']
+    else:
+        raise NotImplementedError(f"Endpoint {url.path} not implemented for "
+                                  "mocking")
+
+
+@pytest.mark.skip("The Mock isn't working yet")
```

Comment:
if you have failures/work in progress things like these, add as a TODO followed by your initials. so if someone later wants to fix/refactor your code, all they need to do is search for TODO. just a suggestion